srcpkgs/spacenavd/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'spacenavd'
+pkgname=spacenavd
+version=1.2
+revision=1
+build_style=gnu-configure
+makedepends="libspnav-devel libX11-devel libXtst-devel"
+depends="libspnav"
```

Comment:
probably autodetected

Reply:
I spent some time digging into the XBPS scripts that do this. Yeah it does seem like it would likely detect this dependency correctly. Do you have any thoughts or preferences on whether any automatically detected dependencies should still be listed here? Should I always omit ones that can be autodetected? I'm asking because you left similar feedback on a different PR (for spnavcfg) so I want to learn the rule and apply it everywhere going forward. I'm tempted to put dependencies that I know up front into this variable as a form of documentation but I'll do whatever you think is best.

Reply:
They should be omitted, the automatically added ones are more precise and generate dependencies based on the minimum version required to provide a specific shared library version.

Reply:
Ok, will do.